convert_color_scheme.py

```diff
@@ -0,0 +1,135 @@
+"""Functionality for converting a color scheme to a "TerminalView" scheme."""
+import os
+import plistlib
+import sublime
+
+
+def hex_to_rgb(hexstring):
+    """Convert a string representing a hex color to an RGB tuple."""
+    return (int(hexstring[1:3], 16) / 255,
+            int(hexstring[3:5], 16) / 255,
+            int(hexstring[5:7], 16) / 255)
+
+
+def rgb_to_hex(rgb):
+    """Convert an RGB tuple to a hex string."""
+    return "#{:02x}{:02x}{:02x}".format(int(rgb[0] * 255),
+                                        int(rgb[1] * 255),
+                                        int(rgb[2] * 255))
+
+
+def norm2(vec3):
+    """Compute the squared norm of a three-dimensional vector."""
+    return vec3[0]**2 + vec3[1]**2 + vec3[2]**2
+
+
+def distance2(a, b):
+    """Compute the squared distance between two 3D vectors."""
+    return norm2((a[0] - b[0], a[1] - b[1], a[2] - b[2]))
```

Comment:
actually, instead of generating the tmTheme file from user file. I guess we should parse the user scheme and save the settings in `TerminalView.sublime-settings`

something like
```
'terminal_color_0'  : '#000000',
'terminal_color_1'  : '#ff8787',
'terminal_color_2'  : '#87d75f',
'terminal_color_3'  : '#ffd75f',
'terminal_color_4'  : '#87afff',
'terminal_color_5'  : '#af87ff',
'terminal_color_6'  : '#d7ff87',
'terminal_color_7'  : '#bfbfbf',
'terminal_color_8'  : '#000000',
'terminal_color_9'  : '#ff8787',
'terminal_color_10' : '#87d75f',
'terminal_color_11' : '#ffd75f',
'terminal_color_12' : '#87afff',
'terminal_color_13' : '#af87ff',
'terminal_color_14' : '#d7ff87',
'terminal_color_15' : '#bfbfbf',
```
and every time when TerminalView is launched, the `hidden-tmTheme` will be created automatically.